Update README and Get Started guide to use accessor (#655)
alteryx/woodwork: README.md

## Example

Below is an example of using Woodwork. In this example, a sample dataset of order items is used to create a Woodwork `DataFrame`, specifying the `LogicalType` for three of the columns.

```python
import woodwork as ww

> df = ww.demo.load_retail(nrows=100, return_dataframe=True)

Reviewer: Can be changed (especially once we rename `load_retail_to_accessor` to `load_retail`)
```suggestion
df = ww.demo.load_retail(nrows=100)
```

> df.ww.init(name='retail')

Reviewer: Let's change `order_product_id` so it is passed in as the index in init

Author: Done

df.ww.set_types(logical_types={

>     'quantity': 'Double',

Reviewer: ```suggestion
    'quantity': 'Integer',
```

Author: Updated

>     'customer_name': 'Categorical',

Reviewer: Is it more appropriate to make `customer_name` a FullName Logical Type?

Author: Yeah, that makes sense. Not sure why we originally went with Categorical, but I'll update.

>     'country': 'Categorical'

Reviewer: Should we add `order_id` to be `Categorical`?

Author: Yes, that makes sense. Changed.

If we wanted to do further analysis on only the columns in this table that have a logical type of `Boolean` or a semantic tag of `numeric` we can simply select those columns and access a dataframe containing just those columns:

```python
filtered_df = df.ww.select(include=['Boolean', 'numeric'])
filtered_df
```

```
    quantity  unit_price   total  cancelled
0          6      4.2075  25.245      False
1          6      5.5935  33.561      False
2          8      4.5375  36.300      False
3          6      5.5935  33.561      False
4          6      5.5935  33.561      False
..       ...         ...     ...        ...
95         6      4.2075  25.245      False
96       120      0.6930  83.160      False
97        24      0.9075  21.780      False
98        24      0.9075  21.780      False
99        24      0.9075  21.780      False
```

> As you can see, Woodwork makes it easy to manage typing information for your data, and provides simple interfaces to access only the data you need based on the logical types or semantic tags. Please refer to the [Woodwork documentation](https://woodwork.alteryx.com/) for more detail on working with Woodwork tables.

Reviewer: ```suggestion
As you can see, Woodwork makes it easy to manage typing information for your data, and provides simple interfaces to access only the data you need based on the logical types or semantic tags. Please refer to the [Woodwork documentation](https://woodwork.alteryx.com/) for more detail on working with a Woodwork DataFrame.
```

Author: Updated
